src/Propel/Generator/Behavior/Validate/ValidateBehavior.php

```diff
@@ -198,19 +198,7 @@ protected function addLoadValidatorMetadataMethod()
                 throw new InvalidArgumentException('Please, define the validator constraint.');
             }
 
-            if (!class_exists("Symfony\\Component\\Validator\\Constraints\\".$properties['validator'], true)) {
-                if (!class_exists("Propel\\Runtime\\Validator\\Constraints\\".$properties['validator'], true)) {
-                    throw new ConstraintNotFoundException('The constraint class '.$properties['validator'].' does not exist.');
-                } else {
-                    $classConstraint = "Propel\\Runtime\\Validator\\Constraints\\".$properties['validator'];
-                }
-            } else {
-                if ($properties['validator'] === 'Unique') {
-                    $classConstraint = "Propel\\Runtime\\Validator\\Constraints\\".$properties['validator'];
-                } else {
-                    $classConstraint = "Symfony\\Component\\Validator\\Constraints\\".$properties['validator'];
-                }
-            }
+            $classConstraint = $this->getClassConstraint($properties);
```

Comment:
Change was introduced by Rene but not pulled into master.

https://github.com/propelorm/Propel2/commit/0dbf800c4ab66ba3616afbff57d61de0a75ad5b3

I re-introduce it for just this part with a co-authored commit.